xwiki-platform-core/xwiki-platform-web/src/main/webapp/templates/informationinline.vm

```diff
@@ -20,6 +21,19 @@
         #end
       </dd>
     </dl>
+    <dl>
+      <dt>$services.localization.render('core.viewers.information.pageReference')</dt>
+      <dd id="information-reference">
+        <span class="noitems">$services.localization.render('core.viewers.information.pageReference.tips')</span>
+        <div class="input-group">
+          <div class="input-group-addon btn btn-default" id="button-inout" title="$services.localization.render('core.viewers.information.pageReference.globalButton')">
+            <span class="fa fa-globe"></span>
+          </div>
+          <span class="form-control" id="reference-value">$doc.getDocumentReference().getLocaleDocumentReference()</span>
```

Comment:
How is a very long reference displayed?

Reply:
Right. I need to check that.

Reply:
Note that I've proposed a solution that could solve this in the jira issue. It's to have UIXP and thus have a tabular layout to support that. See https://jira.xwiki.org/browse/XWIKI-13362?focusedCommentId=92183&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-92183

Reply:
> have a tabular layout

you mean displaying backlinks and included pages in other tabs, right?

Reply:
Nope :) In the comment I linked, there's a screenshot (from jira I think) showing a tabular approach, i.e. one under another and taking the full width.

Now we could still implement it generically (UIXP) and have a grid layout, even if it's harder to implement and may fit less well depending on the UIXs that implement the UIXP.

Reply:
I don't really see how it would fix this issue: if the ref is long enough it would just make an overflow in the table cell. 

Moreover, on another topic I'm not convinced about having a cell with all backlinks and another one with all included pages. I've the feeling it would create an ugly table really long to scroll.